### ISSUE TITLE
Need Login page.

### ISSUE THREAD

**Issue by Hurly77** [Hurly77/Devocidy#29]
We don't have our DB up and running yet. so we just need the data to go anywhere just the design. 
-remember to keep things modular. use MUI-Lib to style and put the container in the Container folders.
- Try to keep thing as dynamic as possible by using the UI/MUI to put any components the may be reusable and then document the changes on the Readme so other people know how to use those components as well.

**Comment by JOEMN96** [comment]
can i create this login page ?

**Comment by Hurly77** [comment]
> can I create this login page?
Yeah, of course, we only need the template. And It needs to have a button for signup that is also a  modal. For now, we don't need the data to go anywhere, just the design for the login.
